pyproject.toml

```diff
@@ -1,3 +1,40 @@
 [build-system]
-requires = ['setuptools >= 40.8.0', 'wheel']
-build-backend = 'setuptools.build_meta'
+requires = ["flit_core >=3.2"]
```

Comment:
3.2 produced incorrect metadata - see https://github.com/pypa/flit/issues/409. Might wanna use `>= 3.3` to be on the safe side.

Reply:
Good point, thanks! I probably need to update that elsewhere too.